version/news/NEWS-2022.02.0-prairie-trillium.md

```diff
@@ -22,9 +22,6 @@
 
 * Added support for setting the `subPath` on Kubernetes sessions using `KubernetesPersistentVolumeClaim` mounts in `/etc/rstudio/launcher-mounts` (Pro #2976).
 * Added support for Slurm 21.08 to the Slurm Launcher plugin
-* Fixed a bug where Slurm Launcher jobs that exited with a non-zero exit code would still have a zero exit code (#203)
-* Fixed a bug where Slurm Launcher jobs with standard error would never be written to the output file (#203)
-* Fixed a bug where Slurm Launcher jobs that exited due to a signal would not show the exit code as 128+signal (#203)
 * Fixed a bug where Launcher log files could be stuck being owned by the root user (#9728)
```

Comment:
Should this be moved to the Fixed heading?

Reply:
Same for :
* Fixed bug with customized display names and launcher sessions (Pro #3217)

Reply:
done